src/navigation/Routes.ios.js

```diff
@@ -411,6 +399,16 @@ function NativeStackNavigator() {
           options={{ customStack: true }}
         />
       )}
+      <NativeStack.Screen
+        component={WalletConnectApprovalSheet}
+        name={Routes.WALLET_CONNECT_APPROVAL_SHEET}
+        {...basicSheetConfig}
+      />
+      <NativeStack.Screen
+        component={WalletConnectRedirectSheet}
+        name={Routes.WALLET_CONNECT_REDIRECT_SHEET}
+        {...basicSheetConfig}
+      />
```

Comment:
I had to do this in order to render a `ContextMenuButton` on top of the Sheet. Under `Stack` it wasn't working.